website/docs/r/cloud_backup_snapshot.html.markdown

```diff
@@ -13,7 +13,12 @@ On-demand snapshots happen immediately, unlike scheduled snapshots which occur a
 
 -> **NOTE:** Groups and projects are synonymous terms. You may find `groupId` in the official documentation.
 
+-> **NOTE:** If you enable a Backup Compliance Policy, no user, regardless of role, can disable the Backup Compliance Policy without contacting MongoDB support, delete a backup snapshot, decrease the retention time for a snapshot after it's taken, disable Cloud Backup, modify the backup policy for an individual cluster below the minimum requirements set in the Backup Compliance Policy, or delete the Atlas project if any snapshots exist. For full list of impacts and more details see [Backup Compliance Policy Prohibited Actions and Considerations](https://www.mongodb.com/docs/atlas/backup/cloud-backup/backup-compliance-policy/#configure-a-backup-compliance-policy).
```

Comment:
```suggestion
-> **NOTE:** If Backup Compliance Policy is enabled for the project for which this backup schedule is defined you cannot delete a backup snapshot or decrease the retention time for a snapshot after it's taken.  See [Backup Compliance Policy Prohibited Actions and Considerations](https://www.mongodb.com/docs/atlas/backup/cloud-backup/backup-compliance-policy/#configure-a-backup-compliance-policy).
```

Reply:
Same comment as above - suggest adding a comma after "....is enabled for the project for which this backup schedule is defined, you cannot... "